work_cycles.md

```diff
@@ -78,6 +78,14 @@ Occurs weekly every Tuesday from 3:30 PM to 4:00 PM. Participants are members of
 1. Ticket any exceptions that should have notified us via monitoring instead of Honeybadger - e.g. Postgres is down.
 1. Look together at any exceptions that are unexpected or may point to us needing to do work - ticket those.
 
+### Team Time
+
+Occurs Tuesdays, Wednesdays, and Thursdays from 11 AM to 1:00 PM and 3:30 PM to 4:30 PM Eastern.
+
+### Goals
```

Comment:
Should this best nested under team time? Does it need its own heading?

Reply:
Yes - not enough `#` - thank you.